runtime/handler.go

```diff
@@ -42,10 +42,11 @@ func ForwardResponseStream(ctx context.Context, mux *ServeMux, marshaler Marshal
 	if d, ok := marshaler.(Delimited); ok {
 		delimiter = d.Delimiter()
 	} else {
-	    delimiter = []byte("\n")
+		delimiter = []byte("\n")
 	}
 
 	var wroteHeader bool
+	ctSet := false
```

Comment:
I'm all for short variable names, but this is too far away from the code using it to use a 2 letter abbreviation IMO. "contentTypeSet" would be preferred

Reply:
done.